### PR TITLE
PCSX2-Core: Change trigger condition for certain console messages 

### PR DESCRIPTION
**Pull request by ssakash**
**Summary of changes:**

* Implement all of willkuer's suggestions mentioned over [here](https://github.com/PCSX2/pcsx2/pull/1266#issuecomment-203932894)
* ~~Prioritize cheats_ws archive over cheats_ws folder for displaying status of widescreen hacks found.~~ ``Removed``
* also make the placement of braces consistent throughout the file.

### CODE REVIEW

**Comment by willkuer** [comment]
I am not sure that prioritizing the zip over the folder is the desired behavior. I think avih implemented that to make it still possible to use manual ws hacking over the automatic one. Maybe this is however not a realistic situation anymore. Most users won't use any manual ws hacks anymore and if they want to use them they can use the cheats folder instead. So I would think either exchanging the priority back or completely removing the ws_cheats folder would make more sense.


If you want to exchange the priority something like that might work:

`
int numberWSHacksLoaded;
bool zipArchiv = (numberWSHacksLoaded = LoadCheatsFromZip(gameCRC, cheats_ws_archive))
if(zipArchiv)
   PatchesCon->WriteLn(Color_Green, "(Wide Screen Cheats DB) Patches Loaded: %d", numberWSHacksLoaded);
else
    numberWSHacksLoaded = LoadCheats(gameCRC, GetCheatsWsFolder(), L"Widescreen hacks"));
gameWsHacks.Printf(L" [%d widescreen hacks]", numberWSHacksLoaded);
`

**Comment by willkuer** [comment]
Btw. what is actually the policy regarding curly brackets? I guess ssakash likes them on a new line. Same for me (who does barely count). But I think for example turtleli likes the opening ones on the same line. So maybe it would be good to have a general rule stick to it? Or just consistent within one file?

**Comment by ssakash** [comment]
> So I would think either exchanging the priority back or completely removing the ws_cheats folder would make more sense.

I vote for removing the ws_cheats folder. It's still possible to edit the patches present on the archive if someone wanted to manually handle the WS patches.


> int numberWSHacksLoaded;
if(!(numberWSHacksLoaded = LoadCheats(gameCRC, GetCheatsWsFolder(), L"Widescreen hacks")))
    if(numberWSHacksLoaded = LoadCheatsFromZip(gameCRC, cheats_ws_archive))
        PatchesCon->WriteLn(Color_Green, "(Wide Screen Cheats DB) Patches Loaded: %d", numberWSHacksLoaded);
gameWsHacks.Printf(L" [%d widescreen hacks]", numberWSHacksLoaded);

Looks better ! I'll use it with additional parenthesis on `` if(numberWSHacksLoaded = LoadCheatsFromZip(gameCRC, cheats_ws_archive))`` to remove a warning.

> Btw. what is actually the policy regarding curly brackets?

 I don't think any kind of rule was enforced on placement of the braces.

**Comment by ssakash** [comment]
Sorry ! thinking again - your suggested code defeats the point of the change on second commit. you'll still print out ``"0 widescreen hacks loaded"`` when calling ``LoadCheats(gameCRC, GetCheatsWsFolder(), L"Widescreen hacks")`` which might mislead some users whereas it doesn't happen on the current second commit due to prioritizing the ws_arhive call before the call to cheats_ws folder.

**Comment by turtleli** [comment]
I don't like the idea of switching the widescreen patch load order or deleting the cheats_ws folder. I think it would make it inconvenient for the people who actually create the patches.

On curly brace placement/style guidelines - There aren't really any guidelines. Personally I try and keep things reasonably consistent at the file level (which can be annoying because I have to check what the file uses). There is a related item on the todo list (astyle), though that would still require choosing what style should be used.

**Comment by ssakash** [comment]
Removed the second commit. ( Needs more opinion , best to handle it in a different PR)

File: pcsx2/gui/AppCoreThread.cpp [hunk 1]
```diff
@@ -406,42 +403,41 @@ void AppCoreThread::ApplySettings( const Pcsx2Config& src )
...
-	}
+	if (EmuConfig.EnableCheats)
+		gameCheats.Printf(L" [%d Cheats]", LoadCheats(gameCRC, GetCheatsFolder(), L"Cheats"));
 
 	// wide screen patches
-	if (EmuConfig.EnableWideScreenPatches) {
-		if (numberLoadedWideScreenPatches = LoadCheats(gameCRC, GetCheatsWsFolder(), L"Widescreen hacks")) {
+	if (EmuConfig.EnableWideScreenPatches)
+	{
+		if ( int numberLoadedWideScreenPatches = LoadCheats(gameCRC, GetCheatsWsFolder(), L"Widescreen hacks"))
```
Comment (turtleli) [inline pcsx2/gui/AppCoreThread.cpp]: There's a space between `(` and `int`.

File: pcsx2/gui/AppCoreThread.cpp [hunk 2]
```diff
@@ -406,42 +403,41 @@ void AppCoreThread::ApplySettings( const Pcsx2Config& src )
...
-		}
-		else {
+		else
+		{
 			// No ws cheat files found at the cheats_ws folder, try the ws cheats zip file.
 			wxString cheats_ws_archive = Path::Combine(PathDefs::GetProgramDataDir(), wxFileName(L"cheats_ws.zip"));
 
-			if (numberDbfCheatsLoaded = LoadCheatsFromZip(gameCRC, cheats_ws_archive)) {
+			if ( int numberDbfCheatsLoaded = LoadCheatsFromZip(gameCRC, cheats_ws_archive))
+			{
```
Comment (turtleli) [inline pcsx2/gui/AppCoreThread.cpp]: Might be unnecessary to use the if/else now.

File: pcsx2/gui/AppCoreThread.cpp [hunk 3]
```diff
@@ -406,42 +403,41 @@ void AppCoreThread::ApplySettings( const Pcsx2Config& src )
...
 		}
 	}
 
 	wxString consoleTitle = gameName + gameSerial;
-	if (!gameSerial.IsEmpty()) {
-		consoleTitle += L" [" + gameCRC.MakeUpper() + L"]";
-	}
-	consoleTitle += gameCompat + gameFixes + gamePatch + gameCheats + gameWsHacks;
-
+	consoleTitle += L" [" + gameCRC.MakeUpper() + L"]" + gameCompat + gameFixes + gamePatch + gameCheats + gameWsHacks;
```
Comment (turtleli) [inline pcsx2/gui/AppCoreThread.cpp]: Maybe combine with previous line?

Comment (ssakash) [inline pcsx2/gui/AppCoreThread.cpp]: I thought of doing it initially but was afraid you'd say something like ``"The line is too wide, I'd like to separate them into two lines"``

Comment (turtleli) [inline pcsx2/gui/AppCoreThread.cpp]: Hmm. Now that I look at it, it is quite long :P

Up to you then whether you split the statement back into two lines or not.

Comment (ssakash) [inline pcsx2/gui/AppCoreThread.cpp]: > Up to you then whether you split the statement back into two lines or not.

I don't mind it either way :P

**Comment by turtleli** [comment]
Well, I suppose I would like that line split into two. And the commit message could be more descriptive. Apart from that, the PR seems fine and I've tested it.

**Comment by ssakash** [comment]
converted them into two lines, also added a more descriptive commit message.

**Comment by turtleli** [comment]
It'd be better if you describe the actual behaviour changes as well in the commit message.

"Always show number of cheats and/or widescreen hacks loaded in console title if the options are enabled" + the other changes you made is better than just "See suggestions provided here - link".

**Comment by ssakash** [comment]
Sorry, it was just me being lazy. I thought you would explain it on your merge commit :P

**Comment by FlatOutPS2** [comment]
> I don't like the idea of switching the widescreen patch load order or deleting the cheats_ws folder. I think it would make it inconvenient for the people who actually create the patches.

I would indeed. It would also become a hassle to use non-16:9 widescreen values, needing to replace all the desired values inside the archive everytime the archive gets an update.

File: pcsx2/gui/AppCoreThread.cpp [hunk 4]
```diff
@@ -406,42 +403,33 @@ void AppCoreThread::ApplySettings( const Pcsx2Config& src )
...
+		gameCheats.Printf(L" [%d Cheats]", LoadCheats(gameCRC, GetCheatsFolder(), L"Cheats"));
 
 	// wide screen patches
-	if (EmuConfig.EnableWideScreenPatches) {
-		if (numberLoadedWideScreenPatches = LoadCheats(gameCRC, GetCheatsWsFolder(), L"Widescreen hacks")) {
+	if (EmuConfig.EnableWideScreenPatches)
+	{
+		if (int numberLoadedWideScreenPatches = LoadCheats(gameCRC, GetCheatsWsFolder(), L"Widescreen hacks"))
 			gameWsHacks.Printf(L" [%d widescreen hacks]", numberLoadedWideScreenPatches);
-		}
```
Comment (turtleli) [inline pcsx2/gui/AppCoreThread.cpp]: The `else` has curly braces, so `if` should use curly braces as well.

Comment (ssakash) [inline pcsx2/gui/AppCoreThread.cpp]: I assume it's some kind of coding convention ? didn't know about it...

**Pull request merged**